hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py

```diff
@@ -845,6 +883,22 @@ def getCurrentPose(self, lname=None, frame_name='WAIST'):
             return HrpsysConfigurator.getCurrentPose(self, lname, frame_name)
     
     def getCurrentPosition(self, lname=None, frame_name='WAIST'):
+        '''!@brief
+        Returns the current physical position of the specified joint.
+        cf. getReferencePosition that returns commanded value.
+
+        eg.
+        \verbatim
+            robot.getCurrentPosition('LARM_JOINT5')
+            [0.325, 0.182, 0.074]
+        \endverbatim
+
+        @type lname: str
+        @param lname: Name of the link.
+        @param frame_name str: set reference frame name (from 315.2.5)
+        @rtype: list of float
+        @return: List of x, y, z positions about the specified joint.
+        '''
```

Comment:
getCurrentPosition calls getCurrentPose internally, so version check is not required.